.clang-format

```diff
@@ -0,0 +1,3 @@
+---
+Language:        Cpp
+BasedOnStyle:  Google
```

Comment:
Please remove that change from this PR. Could be a separate PR.

Reply:
removed